PRs failing on flake8 `DeprecationWarning: SelectableGroups`

Hi,

I'm getting the following deprecation warning from flake8 which is failing some PRs I've put up

> DeprecationWarning: SelectableGroups dict interface is deprecated. Use select.
> [snip]
> FAILED test/test_flake8.py::test_flake8 - DeprecationWarning: SelectableGroup...

As far as I can tell, this is not to do with my changes. I'd infer that something has changed in flake8 and the flake8 validation script needs to be updated.

I've successfully run the `test_flake8` script locally with Python 3.8 and flake8 3.9.2.

Please advise. Thanks.

--- a/colcon_cmake/task/cmake/build.py
+++ b/colcon_cmake/task/cmake/build.py
@@ -64,10 +64,10 @@ class CmakeBuildTask(TaskExtensionPoint):
             action='store_true',
             help='Force CMake configure step')
 
-    async def build(
+    async def build(  # noqa: D102
         self, *, additional_hooks=None, skip_hook_creation=False,
         environment_callback=None, additional_targets=None
-    ):  # noqa: D102
+    ):
         pkg = self.context.pkg
         args = self.context.args
 

--- a/colcon_cmake/test_result/ctest.py
+++ b/colcon_cmake/test_result/ctest.py
@@ -26,9 +26,9 @@ class CtestTestResult(TestResultExtensionPoint):
         satisfies_version(
             TestResultExtensionPoint.EXTENSION_POINT_VERSION, '^1.0')
 
-    def get_test_results(
+    def get_test_results(  # noqa: D102
         self, basepath, *, collect_details, files=None
-    ):  # noqa: D102
+    ):
         results = set()
         # check all 'TAG' files in a directory named 'Testing'
         for tag_file in basepath.glob('**/Testing/TAG'):

--- a/setup.cfg
+++ b/setup.cfg
@@ -51,6 +51,8 @@ zip_safe = true
 [tool:pytest]
 filterwarnings =
     error
+    # Suppress deprecation warning in flake8
+    ignore:SelectableGroups dict interface is deprecated::flake8
 junit_suite_name = colcon-cmake
 
 [options.entry_points]

